File: Flutter App/unicoVehicle/lib/Models/CompanyCountry.dart
import './Country.dart';
import './Miscellaneous/Company.dart';
import './Miscellaneous/District.dart';

class CompanyCountry {
  final int? companyCountryId;
  final Company? company;
  final District? district;
  final Country? country;
  final String? countryHead;
  final bool? isOperational;

  CompanyCountry({
    this.company,
    this.companyCountryId,
    this.country,
    this.countryHead,
    this.district,
    this.isOperational,
  });
}

File: UnicoVehicle/UnicoVehicle/UnicoVehicle.DTO/CompanyCountry.cs
using System;
namespace UnicoVehicle.DTO
{
    public class CompanyCountry
    {
        public int CompanyCountryId { get; set; }
        public Miscellaneous.Company Company { get; set; }
        public Miscellaneous.District District { get; set; }
        public Country Country { get; set; }
        public string CountryHead { get; set; }
        public bool isOperational { get; set; }
    }
}
